wiki/Game_modifier/10K/zh-tw.md

```diff
@@ -0,0 +1,28 @@
+---
+stub: true
+tags:
+  - 10K
+  - mod
+  - game modifier
+  - 遊戲 Mod
+---
+
+# 10K (Mod)
+
+![10K Mod 圖示](/wiki/shared/mods/10K.png "10K Mod 圖示")
+
+*若要查看 Mod 列表，請見：[遊戲 Mod](/wiki/Game_modifier)*\
+*此文章的主題不是 [xK (Mod)](/wiki/Game_modifier/xK)。*
+
+## 關於
+
+- 類型：特殊
+- 適用的遊戲模式：![][osu!mania]
+
+## 說明
+
+*提醒：此 Mod 已無法使用。*
+
+**10K** mod 是 [osu!mania](/wiki/Game_mode/osu!mania) 的 [遊戲 Mod](/wiki/Game_modifier)。讓玩家可以在 osu!mania 的轉譜中使用 10 個按鍵。其功能完全與其他的 [xK](/wiki/Game_modifier/xK) Mod 相同，故亦無法入榜。
```

Comment:
missing link to `/wiki/Beatmap`

Reply:
I skipped the beatmap because translate it would make the sentence looks weird